pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -919,6 +919,29 @@ var _ = Describe("Converter", func() {
 			})
 		})
 
+		Context("CPU mpx feature", func() {
```

Comment:
You could make these tests a table for amd64/arm64/s390x

Reply:
Hi @0xFelix, the amd64 and arm64 will output the mpx feature in the CPU.Features and s390x will output as a nil. So We cannot create a table for all of these as the outputs are different.  May be I can create a table for amd64 and arm64 and a separate testcase for s390x.
Let me know your thoughts on this?

Reply:
Hi @0xFelix I have made the table as you suggested. Can you please review it once again. Thanks